### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by alexmuller**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,10 +4,10 @@ Plek
 "Plek" is Afrikaans. It means "Location." Plek is used to generate the correct hostnames for internal GOV.UK services, eg:
 
 ```ruby
-Plek.find('frontend')
+Plek.current.find('frontend')
```
Comment (alext) [inline README.md]: Why this change?  `Plek.find` is a valid method that works.

Comment (alexmuller) [inline README.md]: Ah yes, sorry. I was using an old version of Plek inside one of our applications and the method didn't exist. I'll remove this commit.
